bowler/tool.py

```diff
@@ -143,10 +142,12 @@ def processed_file(
                 hunks.append([a, b, *hunk])
 
             try:
-                ast.parse(new_text, filename)
+                new_tree = self.driver.parse_string(new_text)
+                if new_tree is None:
+                    raise AssertionError("Re-parsed CST is None")
```

Comment:
Is there a specific case where this would happen?  Should we raise a different/new `BowlerError` instead of an assert?

Reply:
On second look, I think the case I was checking for is in `_read_python_source` for nonexistent files, not something that `parse_string` can do.  (The only way for this to be None would be a bug in the state machine of pgen2.)

I'm fine with AssertionError for now, this shouldn't be a common case.